x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -15,6 +16,8 @@ import { CSP_RULE_SAVED_OBJECT_TYPE } from '../constants';
 export const isNonNullable = <T extends unknown>(v: T): v is NonNullable<T> =>
   v !== null && v !== undefined;
 
+export const truthy = <T>(value: T): value is Truthy<T> => !!value;
```

Comment:
wanted to filter out undefined values from an array, but apparently, typescript doesn't work too well with `filter(Boolean)` and recognizes the array as `false | <some item>` even though the filter should have cleared any false/falsy items.